SnapcallParameter.js

```diff
@@ -0,0 +1,59 @@
+import { Platform, Image } from 'react-native';
+
+
+let os = Platform.OS === "ios" ? true : false;
+/**
+    class with the differente parameter to configure the call UI and the data you send.
+**/
+
+function getImage(image) {
+  const resolveAssetSource = require('react-native/Libraries/Image/resolveAssetSource');
+  if (resolveAssetSource){
+    return resolveAssetSource(image);
+  }
+  return null;
+}
+
+export class SnapcallParameter {
+
+  transformJsonParameter(obj) {
```

Comment:
What do you think about this approach?

- You can iterate over the keys straight away with a `forEach`.
- You can also save the `if` since `Object.keys` return an empty array if `obj` is `undefined`
- I have a question, why do you need to `stringify` true and false? (I'm guessing this is related to iOS and Android?)

```
transformJsonParameter(obj) {
  const snapcallParameter = {};
  Object.keys(obj).forEach((key) => {
    if (os && obj[key]) {
      snapcallParameter[key] = (obj[key]) ? 'true' : 'false';
    } else {
      snapcallParameter[key] = obj[key];
    }
  });
  return JSON.stringify(snapcallParameter);
}
```

Or if you prefer `.reduce`
```
const snapCallParameter = Object.keys(obj).reduce((acc, curr) => {
    // do your checks
}, {});

return JSON.stringify(snapCallParameter);
```

Reply:
Hey i removed this part has no more necessary thanks for your feedback. 
This was because in obj-c true is not an object and can't be into a nsdictionnary(~> json object ) , now i use correctly object representation  and every thing is fine.